test/katex-spec.js

```diff
@@ -1212,6 +1258,19 @@ describe("A left/right parser", function() {
     });
 });
 
+describe("left/right builder", () => {
+    const cases = [
+        ['\\left\\langle \\right\\rangle', '\\left< \\right>'],
+        ['\\left\\langle \\right\\rangle', '\\left\u27e8 \\right\u27e9'],
+    ];
```

Comment:
I think this could be extend to something like:
```
const cases = {
    'shouldParse': [ ... ],
    'shouldBuild': [ ... ],
    'shouldNotParse': [ ... ],
    'shouldNotBuild': [ ... ],
    'shouldParseLike': [ ... ],
    'shouldBuildLike': [ ... ],
};
```
I'll open an issue for this.